docs/reference/go-style-annotation-property-practice.md

```diff
@@ -0,0 +1,16 @@
+# Recommended metadata properties to use in curating OBO ontologies (GO-style)
+
```

Comment:
Great table! I would:

- dcterms instead of dc in all cases
- add oboInOwl prefix to all synonym types
- add owl:deprecated and replaced by relations